src/com/google/enterprise/adaptor/opentext/OpentextAdaptor.java

```diff
@@ -901,9 +917,16 @@ void doContainer(DocumentManagement documentManagement,
       containerContents = documentManagement.listNodes(containerNode.getID(),
           true);
     } catch (SOAPFaultException soapFaultException) {
-      log.log(Level.WARNING,
-          "Error retrieving container contents: " + opentextDocId,
-          soapFaultException);
+      if (isPermissionsFailure(soapFaultException)) {
+        log.log(Level.FINE,
+            soapFaultException.getFault().getFaultString() + " {0}",
```

Comment:
I think use MessageFormat-style or not: "{0} {1}" or soapFaulException.... + opentextDocId. toString() is redundant again here. Maybe a "for" would not go amiss in there?